src/app/app.component.ts

```diff
@@ -35,7 +35,8 @@ export class AppComponent {
       this.snackBar.open(message, 'Dismiss');
     });
     this.notificationService.error$.subscribe(message => {
-      this.dialog.open(ModalComponent, {data: message});
+      const dismissInfo = ' Click anywhere to dismiss.';
```

Comment:
nit: leading space in the message.

Reply:
The leading space is because the message it is appended to will end with a "."
Let me know if there is a better/more readable way to do this.

Reply:
Can we just have a "close" button, rather than editing the message?

Reply:
Yes, I didn't realise just how easy that is to implement with angular.